.github/workflows/container.yaml

```
@@ -41,14 +41,12 @@ jobs:
           set -x
           releaseBranchFormat='release-v'
           if [[ ${{ github.ref_name }} == ${releaseBranchFormat}* ]]; then
-            tag="${${{ github.ref_name }}#*-}" 
+            tag=$(echo ${{ github.ref_name }}|sed "s,${releaseBranchFormat},,")
           elif [[ ${{ github.ref }} == refs/pull/* ]]; then
```

shouldn't it be "refs/tags/*"?

```suggestion
          elif [[ ${{ github.ref }} == refs/tags/* ]]; then
```